### DISCUSSION TITLE
Some forms lagging behind or inserting extra text.

### DISCUSSION THREAD

**Discussion by ChaosNicro** [AnySoftKeyboard/AnySoftKeyboard#3784] [Q&A]
Hey there,
I'm sure there is alredy a topic about this, but I couldn't find any.
When deleting text from a github issue template for example, I get a de-sync, where parts of the text continuously reappear while holding down backspace.
Or in my notes app "my brain"
Hitting the enter key for a newline will create a copy of the last typed word on the new line.
I'm sure this is just text-fields wanting to offer live-typing-features and lagging behind the keyboard input. Is there a way to mitigate this de-sync behaviour? Is it a problem with the specific text-field or ASK?
I would think that Github would be to big to just be badly optimized.

Thanks in advance.

**Comment by einkleinerwolf** [comment]
Lately I'm having this issue nearly everywhere.
Is there any known reason or fix yet?

**Comment by ChaosNicro** [comment]
Nope, but it's equivalent to #4448 . 
So any discussion should happen there.
